airflow/config_templates/config.yml

```diff
@@ -28,6 +28,14 @@ core:
       type: string
       example: ~
       default: "{AIRFLOW_HOME}/dags"
+    dag_bundle_storage_path:
+      description: |
+        The folder where Airflow will store DAG bundles locally.
+        By default, this will be ``tempfile.gettempdir()/airflow``. This path must be absolute.
```

Comment:
i think this needs to be updated. logic looks different.

Reply:
Updated.